packages/gatsby-transformer-sharp/src/gatsby-node.js

```diff
@@ -1,32 +1,14 @@
 const fs = require(`fs-extra`)
 
 exports.onCreateNode = require(`./on-node-create`)
-exports.setFieldsOnGraphQLNodeType = require(`./extend-node-type`)
+exports.createSchemaCustomization = require(`./customize-schema`)
 
-exports.onPreExtractQueries = async ({ store, getNodesByType }) => {
+exports.onPreExtractQueries = async ({ store }) => {
   const program = store.getState().program
 
-  // Check if there are any ImageSharp nodes. If so add fragments for ImageSharp.
-  // The fragment will cause an error if there are no ImageSharp nodes.
-  if (getNodesByType(`ImageSharp`).length == 0) {
-    return
-  }
-
-  // We have ImageSharp nodes so let's add our fragments to .cache/fragments.
+  // Add fragments for ImageSharp to .cache/fragments.
   await fs.copy(
```

Comment:
This is no longer necessary as we run babel on plugin code now. We should be able to remove the copy operation.